gptresearcher/context/llm_utils.py

```diff
@@ -3,30 +3,25 @@
 import json
 
 from fastapi import WebSocket
-import time
 
 import openai
 from langchain.adapters import openai as lc_openai
 from colorama import Fore, Style
-from openai.error import APIError, RateLimitError
 
-from agent.prompts import auto_agent_instructions
+from gptresearcher.context.prompts import auto_agent_instructions
 from config import Config
 
-CFG = Config()
-
-openai.api_key = CFG.openai_api_key
-
 from typing import Optional
-import logging
+
 
 def create_chat_completion(
```

Comment:
@gregdrizz you can see here an example of passing many params instead of an object